src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -244,11 +244,14 @@ const DOCS: {[key: string]: DocCategory[]} = {
       items: [
         {id: 'a11y', name: 'Accessibility', examples: []},
         {id: 'bidi', name: 'Bidirectionality', examples: []},
+        {id: 'coercion', name: 'Coercion', examples: []},
```

Comment:
Lets leave `coercion` out for now; I'd like to revisit it if/when we add more stuff

Reply:
Done.